presto-docs/src/main/sphinx/release/release-0.239.rst

```diff
@@ -17,8 +17,7 @@ Release 0.239
 
 General Changes
 _______________
-* Fix incorrect results from :func:`classification_miss_rate`, :func:`classification_fall_out`
-  and :func:`classification_precision` (:pr:`14740`).
+* Fix incorrect results from :func:`classification_miss_rate`, :func:`classification_fall_out` (:pr:`14740`).
```

Comment:
Please add a warning section at the top similar to https://github.com/prestodb/presto/pull/15048/files#diff-caa0a1e2c634de5f8adf9254bd7b4850R5-R8
This prevent open source users from using this release, otherwise it is not explicit.